mycroft/tts/espeak_tts.py

```diff
@@ -32,8 +32,32 @@ def get_tts(self, sentence, wav_file):
         Returns:
             tuple ((str) file location, None)
         """
-        subprocess.call(['espeak', '-v', self.lang + '+' + self.voice,
-                         '-w', wav_file, sentence])
+
+        # Create Argument String for Espeak
+        arguments = ['espeak', '-v', self.lang + '+' + self.voice]
+        argument = self.config.get("amplitude")
+        if argument:
```

Comment:
I know it's not a complex function, however I think we can make it clearer by using the name of each argument as the variable names eg:
```
amplitude = self.config.get...
if amplitude:
  ...
gap = ...
```

Reply:
Thanks for the Feedback :)
Improved it